runners/direct-java/pom.xml

```diff
@@ -247,6 +247,14 @@
   </build>
 
   <dependencies>
+    <!-- jsr305 dependency must come before java-sdk-all, as java-sdk-all depends on
```

Comment:
@dhalperi and I fought with this for a while yesterday. The issue is that both `com.google.code.findbugs:jsr305` and `com.google.code.findbugs:annotations` artifacts contain `javax.annotation.Nullable` (and other annotations) [[1]](https://repository.sonatype.org/#nexus-search;quick~com.google.code.findbugs). We compile against whichever is found first on the classpath, and if that ends up being a different JAR than the declared dependency, the dependency analyzer fails.

Other possible solutions:

1. Don't use `com.google.code.findbugs:annotations`. This is used for @SuppressFBWarnings annotation to add FindBugs suppressions inline. We could instead put all of our suppressions in the XML file. That solution is worse because it breaks code locality.

2. Don't use `com.google.code.findbugs:jsr305` and instead reference `com.google.code.findbugs:annotations` throughout codebase. This may fix the immediate issue, but I believe we would hit the same issue if another dependency transitively depends on `findbugs:jsr305` and is declared before `findbugs:annotations`.